ciBuildDocker.sh

```diff
@@ -5,7 +5,7 @@ set -e
 version=`echo ${APPLICATION_VERSION} | sed 's/[^a-zA-Z0-9-]/\_/g' | awk '{print tolower($0)}'`
```

Comment:
I wonder if we really need this file.. We can move all these logics to sbt and use it with envs..

Reply:
Maybe we don't but this change is conservative one (I don't have ability to deeply test it).